packages/xrpl/tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "pretty": true,
-    "target": "es6",
+    "target": "ES2022",
```

Comment:
Why did this increase? (And is it possible to keep it lower? - It appears to be a potentially breaking change - https://stackoverflow.com/questions/73387445/what-happens-if-tsconfig-targets-and-libs-are-mixed-among-dependencies-for-t )

Reply:
Fixed